src/Microsoft.ApiDesignGuidelines.Analyzers/Core/ImplementIDisposableCorrectly.cs

```diff
@@ -14,6 +17,9 @@ public abstract class ImplementIDisposableCorrectlyAnalyzer : DiagnosticAnalyzer
     {
```

Comment:
I made the class sealed and removed C# and VB classes. However, I added suppress warning for RS1008, because of issue #845.